pages/search-site/index.vue

```diff
@@ -0,0 +1,327 @@
+<script setup>
+// UTILITIES
+import getListingFilters from '../utils/getListingFilters'
+import config from '../utils/searchConfig'
+import queryFilterHasValues from '../utils/queryFilterHasValues'
+
+const route = useRoute()
+const page = ref({})
+const from = ref(0)
+const previous = ref(false)
+const next = ref(false)
+const prevFrom = ref(0)
+const nextFrom = ref(0)
+const size = ref(10)
+const noResultsFound = ref(false)
+const searchFilters = ref([])
+const searchGenericQuery = ref({
+  queryText: route.query.q || '',
+  queryFilters:
+    (route.query.filters &&
+      JSON.parse(route.query.filters)) ||
+    {},
+})
+const isSearching = ref(true)
+const { $dataApi } = useNuxtApp()
+// This watcher is called when router push updates the query params
+watch(
+  () => route.query,
+  (newVal, oldVal) => {
+    console.log('Site search page ES newVal, oldVal', newVal, oldVal)
```

Comment:
I'm wondering if we want to comment out some of these console logs? The console is very noisy

Reply:
I want to keep it for now and will comment it once it is merged and tested on nuxt3.x netlify environment.